server/helpers/utils.js

```diff
@@ -0,0 +1,15 @@
+import slug from 'slug';
+import uuid from 'uuid/v4';
+
+/**
+ * @param  {string} title
+ * @return {string} unique_slug
+ */
+const createUniqueSlug = title => `${slug(title, { lower: true })}-${Date.now()}`;
+
+/**
+ * @return {string} unique_uuid
+ */
+const createUniqueId = () => uuid();
```

Comment:
This does not save you any code, there is no need to abstract everything. This library is already abstracted.